### PR TITLE
Prevents GitHub Actions to be executed by forks

### PR DESCRIPTION
**Pull request by abdonrd**
Also:
- Extract the node version to a env variable.
- Clean the naming to better read:
  Screenshot 2022-01-17 at 10 53 58

### CODE REVIEW

**Review by Tansito — DISMISSED** [review]
LGTM!

File: .github/workflows/preview-cleanup.yml [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
-# run when a pull request is closed
 on:
   pull_request:
     types: [closed]
 
 jobs:
   code-engine:
+    # Run the job if it's not triggered by a PR or if the PR is from the same repository
+    # This is because this job requires a secret, and secrets cannot be shared with forks
+    if: github.event.pull_request == null || github.event.pull_request.head.repo.full_name == github.repository
```
Comment (vabarbosa) [inline .github/workflows/preview-cleanup.yml:11]: for my own curiosity, if this is set to only run`on: pull_request` when would `github.event.pull_request == null`?

File: .github/workflows/preview-deploy.yml [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
+on:
+  pull_request:
 
 jobs:
   code-engine:
-    # Prevents action to be executed by forks
-    if: github.event.pull_request.head.repo.full_name == github.repository
+    # Run the job if it's not triggered by a PR or if the PR is from the same repository
+    # This is because this job requires a secret, and secrets cannot be shared with forks
+    if: github.event.pull_request == null || github.event.pull_request.head.repo.full_name == github.repository
```
Comment (vabarbosa) [inline .github/workflows/preview-deploy.yml:10]: same question as above, if this is set to only run`on: pull_request` when would `github.event.pull_request == null`?

**Review by vabarbosa — APPROVED** [review]
thanks. looks good... just had one question

File: .github/workflows/preview-cleanup.yml (hunk 1 above)
Comment (abdonrd) [reply .github/workflows/preview-cleanup.yml:11]: You're right! No needed in this case!

**Review by eddybrando — APPROVED** [review]
LGTM! Thanks, @abdonrd!

**Pull request merged**